users/views.py

```
@@ -1010,5 +1011,20 @@ class FetchUserAnalytics(ClientProtectedResourceMixin, View):
 
     def get(self, request, *args, **kwargs):
-        users = ConnectUser.objects.filter(is_active=True).values("username", "hq_sso_date")
+        users = ConnectUser.objects.filter(is_active=True, is_staff=False).values("username", "hq_sso_date")
         return JsonResponse({"data": list(users)})
+
+
+class AddUserAnalytics(APIView):
+    """This view gets called by CommCareHQ to add user analytics  for
+    a PersonalID User."""
+
+    authentication_classes = [ClientProtectedResourceAuth]
```

Ruff (0.14.7)

Mutable class attributes should be annotated with `typing.ClassVar`

(RUF012)